ArduSub/GCS_MAVLink_Sub.cpp

```diff
@@ -219,7 +219,7 @@ void GCS_MAVLINK_Sub::send_pid_tuning()
         }
     }
     if (g.gcs_pid_mask & 8) {
-        const AP_PIDInfo &pid_info = sub.pos_control.get_accel_U_pid().get_pid_info();
+        const AP_PIDInfo &pid_info = sub.pos_control.D_get_accel_pid().get_pid_info();
```

Comment:
@Williangalvani, FYI, we're renaming some position controller methods so this PR affects sub as well.  I think the changes are fairly straightforward for sub though